tests_pytest/keras/gptq/test_gradual_act_quantization.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2024 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from unittest.mock import Mock
+import pytest
+import numpy as np
+import tensorflow as tf
+
+from model_compression_toolkit.gptq.common.gradual_activation_quantization import GradualActivationQuantizerWrapper, \
+    get_gradual_activation_quantizer_wrapper_factory
+from model_compression_toolkit.trainable_infrastructure.keras.annealing_schedulers import KerasLinearAnnealingScheduler
+from model_compression_toolkit.gptq import GradientPTQConfig, GradualActivationQuantizationConfig, QFractionLinearAnnealingConfig
+
+
+
+@pytest.fixture
+def x():
+    return tf.random.normal((2, 5, 6, 7), seed=42, dtype=tf.float32)
+
+
+class Quantizer:
+    def __call__(self, x, training):
+        self.training = training
+        return 3 * x + 1
+
+
+class TestGradualActivationQuantization:
+
+    def test_gradual_act_quant_wrapper(self, x):
+        quantizer = Quantizer()
+        qw = GradualActivationQuantizerWrapper(quantizer, q_fraction_scheduler=lambda t: t / (t + 1))
+
+        y0, y1, y2 = [qw(x, training=True) for _ in range(3)]
+        np.testing.assert_array_almost_equal(y0.numpy(), x.numpy())  # t=0
```

Comment:
Why np.testing? is there any advantages of using this tool?
unless there is a clear advantage I think it is preferred that we'll be aligned with the assertion methodology that Irena used for the Pytorch Pytests (which is, to use Python's built in assertion).